tox.ini

```diff
@@ -25,26 +25,23 @@ pass_env =
 
 [testenv:format]
 description = Apply coding style standards to code
-deps =
-    black
-    ruff
-commands =
-    black {[vars]all_path}
+deps = ruff
+commands = 
     ruff format {[vars]all_path}
+    ruff check --fix {[vars]all_path}
 
 [testenv:lint]
 description = Check code against coding style standards
 deps =
-    black
     ruff
+    tomli
```

Comment:
@mateoflorido i think when i removed black, i took `tomli` with it.  Because `tomli` was gone in python 3.12 -- i think codespell refused to read the pyproject.toml.  

Adding `tomli` back in -- has everything work as it should